critiquebrainz/frontend/forms/review.py

```diff
@@ -20,13 +21,32 @@ def __call__(self, form, field):
             raise ValidationError(self.message)
 
 
-# Loading supported languages
-languages = []
-for language_code in supported_languages:
+OTHER_LANGUAGES_CODES = supported_languages
```

Comment:
Put this code in a function
```
def get_language_choices():
    
    other_language_codes = supported_languages
    frequently_used_languages_codes = get_top_languages()
    for language in frequently_used_languages_codes:
        try:
            other_language_codes.remove(language)
        except ValueError:
            continue
    other_languages = []
    frequently_used_languages = []
    for language_code in other_language_codes:
        try:
            other_languages.append((language_code, Locale(language_code).language_name))
        except UnknownLocaleError:
            other_languages.append((language_code, pycountry.languages.get(iso639_1_code=language_code).name))
    for language_code in frequently_used_languages_codes:
        try:
            frequently_used_languages.append((language_code, Locale(language_code).language_name))
        except UnknownLocaleError:
            frequently_used_languages.append((language_code, pycountry.languages.get(iso639_1_code=language_code).name))
    
    languages = (('Frequently Used Languages', tuple(frequently_used_languages)),
                 ('Other Languages', tuple(other_languages)))
    return languages
```
I guess, we can later improve this by defining a script for frequent languages (as in MusicBrainz).